x-pack/plugins/reporting/server/routes/internal/generate/csv_searchsource_immediate.ts

```diff
@@ -9,19 +9,16 @@ import Boom from '@hapi/boom';
 import { schema } from '@kbn/config-schema';
 import type { KibanaRequest, Logger } from '@kbn/core/server';
 import moment from 'moment';
-import type { ReportingCore } from '../..';
-import { CSV_SEARCHSOURCE_IMMEDIATE_TYPE } from '../../../common/constants';
-import { JobParamsDownloadCSV } from '../../export_types/csv_searchsource_immediate/types';
-import { PassThroughStream } from '../../lib';
-import { authorizedUserPreRouting, getCounters } from '../lib';
+import type { ReportingCore } from '../../..';
+import { CSV_SEARCHSOURCE_IMMEDIATE_TYPE, INTERNAL_ROUTES } from '../../../../common/constants';
+import type { JobParamsDownloadCSV } from '../../../export_types/csv_searchsource_immediate/types';
+import { PassThroughStream } from '../../../lib';
+import { authorizedUserPreRouting, getCounters } from '../../common';
 
-const API_BASE_URL_V1 = '/api/reporting/v1';
-const API_BASE_GENERATE_V1 = `${API_BASE_URL_V1}/generate`;
+const path = INTERNAL_ROUTES.DOWNLOAD_CSV;
```

Comment:
I love that there's the constants/routes file in this PR to reference these vs it being spread across the plugin